docs/conf.py

```diff
@@ -41,6 +41,7 @@
     # 'sphinx.ext.graphviz',
     'IPython.sphinxext.ipython_console_highlighting',
     'IPython.sphinxext.ipython_directive',
+    'sphinx.ext.autosectionlabel',
```

Comment:
Nice extension! I didn't know about that, but it definitely looks useful. Thanks!